content/collections/account-management/en/self-service-data-deletion-in-amplitude.md

```diff
@@ -0,0 +1,45 @@
+---
+id: 1b48f869-0e38-45ac-a6d5-03184a2ac77d
+blueprint: account-management
+title: 'Self-service data deletion in Amplitude'
+this_article_will_help_you:
+  - 'Delete data from Amplitude permanently'
+landing: false
+exclude_from_sitemap: false
+updated_by: 5817a4fa-a771-417a-aa94-a0b1e7f55eae
+updated_at: 1728428160
+---
+Sometimes, you may need to permanently remove data from your Amplitude projects in a timely manner. For example, maybe your product sent incorrect data to Amplitude last month, but that data has since been corrected, and you’d like to remove the original, incorrect events or properties.
+
+Amplitude’s **self-service data deletion** feature lets you do this, without involving Amplitude personnel. Simply specify which events or properties you want to delete and send your deletion request to the Amplitude deletion queue.
+
+## Before you begin
+
+This feature is available to users on Growth and Enterprise plans only. Growth users get one deletion task, to include up to one billion events scanned, per month. Enterprise users get 10 tasks, each of which can include as many as one billion events scanned, per month.
+
+Only users with Admin permissions can use this feature.
+
+**You can’t cancel your request** to delete data once your submitted request is in the Amplitude deletion queue.
```

Comment:
```suggestion
Once you submit your deletion request to Amplitude, you can't cancel it or remove it from the queue.
```

I think this sentence might read better if it's written chronologically. It might also deserve a warning or other admonition.